datacenter/ucp/2.0/guides/release-notes.md

```diff
@@ -10,6 +10,34 @@ known issues for the latest UCP version.
 You can then use [the upgrade instructions](installation/upgrade.md), to
 upgrade your installation to the latest release.
 
+## Version 2.0.3
+
+(8 Feb 2017)
+
+**Bug Fixes**
+
+* Core
+  * Added role-based access control for volumes
+  * Demoting a manager no longer causes problems in the UCP authentication service
```

Comment:
A bit vague. Probably should err on the side of being specific and say "Demoting a manager while in HA configuration no longer causes `ucp-auth-api` container to provide errors"